modules/understanding-agent-install.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent/installing-with-agent.adoc
+
+:_content-type: CONCEPT
+[id="understanding-agent-install_{context}"]
+= Understanding Agent installation
+As a {product-title} user, you can leverage all the advantages of the Assisted Installer hosted service in disconnected environments. You can generate an ISO that contains all the technology required to run both a user friendly guided deployment and a fully automated one.
+The Agent installation comprises of a bootable ISO that contains the Assisted discovery agent and the Assisted service; both are required to perform the first cluster installation.
+
+The `openshift-install agent create` subcommand generates an ephemeral ISO that is generated based on the inputs that you provide in the following manifests:
+
+* `ClusterImageSet`
+* `ClusterDeployment`
+* `pull-secret`
+* `InfraEnv`
+* `AgentClusterInstall`
+* `NMStateConfig`
+
+//TODO workflow diagram
+
+You can install a disconnected {product-title} cluster through the `openshift-install agent create image` subcommand for:
+
+* **A single-node {product-title} cluster (SNO)**: A node that is both a master and worker.
+* **A three-node {product-title} cluster** : A compact cluster that has three master nodes that are also worker nodes.
+* **Highly available {product-title} cluster (HA)**: Three master nodes with any number of worker nodes.
+
+The following platforms are supported:
+
+* none
+* bare-metal
+* vSphere
+
+[id="about-static-networking"]
+== About static networking
+
+You need to provide one static IP configuration for `node0`, which is the node that runs the installation backend and will eventually be a part of `node0`.
```

Comment:
@bfournie Does the user need to provide the nmstate settings to configure the host with a static IP, or just the IP address that host will get via DHCP?